libraries/iio/iio_types.h

```diff
@@ -161,6 +167,8 @@ struct iio_channel {
 	 *  for angular rate when applied to channel2 will make make the
 	 *  IIO_ANGL_VEL have anglvel_x which corresponds to the x-axis. */
 	int 			channel2;
+	/** Driver specific identifier. */
+	unsigned long		address;
```

Comment:
Is the linux naming?
Doesn't it make more sense to be void *context or something like that?

Reply:
i don't think changing names will improve anything - it will actually make it worse, every time you wanna port an iio driver implementation between linux <-> no-OS you need to think about which corresponds to which